### PR TITLE
Target audience is developers

### PR DESCRIPTION
**Pull request by RCdiy**
I will add a section in the manual for users.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 [[![Join the chat at https://opentx.rocket.chat](https://camo.githubusercontent.com/3d659054abd6ce21c0e47cf3b83a51bda69ca282/68747470733a2f2f64656d6f2e726f636b65742e636861742f696d616765732f6a6f696e2d636861742e737667)](https://opentx.rocket.chat)]()
 
+Target Audience: Developers
+
```
Comment (mpaperno) [inline README.md]: How about something like:

This repository is meant for documentation editors.

**Go to https://www.gitbook.com/book/opentx/opentx-lua-reference-guide/details for the latest published version of this guide.**

File: SUMMARY.md [hunk 2]
```diff
@@ -1,120 +1,121 @@
...
-      * [lcd.drawTimer(x, y, value [, flags])](lcd/drawTimer.md)
-      * [lcd.getLastPos()](lcd/getLastPos.md)
-      * [lcd.refresh()](lcd/refresh.md)
-      * [lcd.setColor(area, color)](lcd/setColor.md) [//]: <> (LUADOC-END:lcd)
-   * [Bitmap Functions](Bitmap/Bitmap_functions.md) [//]: <> (LUADOC-BEGIN:Bitmap)
-      * [Bitmap.getSize(name)](Bitmap/getSize.md)
-      * [Bitmap.open(name)](Bitmap/open.md) [//]: <> (LUADOC-END:Bitmap)
+    * [Constants](constants.md)
+        * [Key Event Constants](key_events.md)
+    * [General Functions](general/general_functions.md)
```
Comment (projectkk2glider) [inline SUMMARY.md]: Careful here! The parts of this file are generated by a special script that parses OpenTX source files and generates documentation. 

This script needs `[//]: <> (LUADOC-BEGIN:general)` - this is a special marker for  it. It won't be visible in final book, but script searches for it.

File: SUMMARY.md [hunk 3]
```diff
@@ -1,120 +1,121 @@
...
+    * [Output Table Syntax](output_table_syntax.md)
+    * [Init Function Syntax](init_function_syntax.md)
+    * [Run Function Syntax](run_function_syntax.md)
+    * [Return Statement Syntax](return_statement_syntax.md)
+    * [Included Lua Libraries](included_lua_libraries.md)
+        * [io Library](lib/io.md)
+            * [io.open\(\)](lib/io_open.md)
+            * [io.close\(\)](lib/io_close.md)
+            * [io.read\(\)](lib/io_read.md)
+            * [io.write\(\)](lib/io_write.md)
```
Comment (projectkk2glider) [inline SUMMARY.md]: Why did you escape `()` characters?

File: README.md (hunk 1 above)
Comment (kilrah) [inline README.md]: This will appear on the book, not just in the repo. Intent is for the viewer to see the book targets script developers.

Comment (mpaperno) [inline README.md]: Oh.  My point still stands then, which developers are the targets?

**The target audience of this reference guide is Lua script developers.**

Perhaps also with a link to end-user docu if/when exists.

Comment (mpaperno) [inline README.md]: And if anyone is wondering... "**is** Lua script developers" and "**are** Lua script developers" are both correct... just depends on which English you are used to.  😄

**Comment by RCdiy** [comment]
Correct, the pull request mentioned that I want to add a Lua section in the OpenTx manual. That section will be targeted to users who receive a script form somewhere.

On Jan 16, 2017, at 4:21 PM, Andre Bernet <notifications@github.com> wrote:

@kilrah commented on this pull request.

In README.md <https://github.com/opentx/lua-reference-guide/pull/26>:

> @@ -2,6 +2,8 @@
 
 [[Join the chat at https://opentx.rocket.chat](https://opentx.rocket.chat)]()
 
+Target Audience: Developers
+
This will appear on the book, not just in the repo. Intent is for the viewer to see the book targets script developers.

**Comment by RCdiy** [comment]
That is what GitBook did, may be they updated how they code the books?

Looks okay on my gitbook.

On Jan 16, 2017, at 4:18 PM, Damjan Adamic <notifications@github.com> wrote:

@projectkk2glider requested changes on this pull request.

In SUMMARY.md <https://github.com/opentx/lua-reference-guide/pull/26#pullrequestreview-16884216>:

> -      * [lcd.drawPoint(x, y)](lcd/drawPoint.md)
-      * [lcd.drawRectangle(x, y, w, h [, flags [, t]])](lcd/drawRectangle.md)
-      * [lcd.drawScreenTitle(title, page, pages)](lcd/drawScreenTitle.md)
-      * [lcd.drawSource(x, y, source [, flags])](lcd/drawSource.md)
-      * [lcd.drawSwitch(x, y, switch, flags)](lcd/drawSwitch.md)
-      * [lcd.drawText(x, y, text [, flags])](lcd/drawText.md)
-      * [lcd.drawTimer(x, y, value [, flags])](lcd/drawTimer.md)
-      * [lcd.getLastPos()](lcd/getLastPos.md)
-      * [lcd.refresh()](lcd/refresh.md)
-      * [lcd.setColor(area, color)](lcd/setColor.md) [//]: <> (LUADOC-END:lcd)
-   * [Bitmap Functions](Bitmap/Bitmap_functions.md) [//]: <> (LUADOC-BEGIN:Bitmap)
-      * [Bitmap.getSize(name)](Bitmap/getSize.md)
-      * [Bitmap.open(name)](Bitmap/open.md) [//]: <> (LUADOC-END:Bitmap)
+    * [Constants](constants.md)
+        * [Key Event Constants](key_events.md)
+    * [General Functions](general/general_functions.md)
Careful here! The parts of this file are generated by a special script that parses OpenTX source files and generates documentation.

This script needs [//]: <> (LUADOC-BEGIN:general) - this is a special marker for it. It won't be visible in final book, but script searches for it.

In SUMMARY.md <https://github.com/opentx/lua-reference-guide/pull/26#pullrequestreview-16884216>:

> -           * [io.open()](lib/io_open.md)
-           * [io.close()](lib/io_close.md)
-           * [io.read()](lib/io_read.md)
-           * [io.write()](lib/io_write.md)
-           * [io.seek()](lib/io_seek.md)
+    * [Input Table Syntax](input_table_syntax.md)
+    * [Output Table Syntax](output_table_syntax.md)
+    * [Init Function Syntax](init_function_syntax.md)
+    * [Run Function Syntax](run_function_syntax.md)
+    * [Return Statement Syntax](return_statement_syntax.md)
+    * [Included Lua Libraries](included_lua_libraries.md)
+        * [io Library](lib/io.md)
+            * [io.open\(\)](lib/io_open.md)
+            * [io.close\(\)](lib/io_close.md)
+            * [io.read\(\)](lib/io_read.md)
+            * [io.write\(\)](lib/io_write.md)
Why did you escape () characters?

**Pull request closed without merging**